source/Octopus.Versioning/Maven/MavenPackageID.cs

```diff
@@ -147,7 +147,7 @@ public MavenPackageID([NotNull]
         }
 
         public string Group { get; }
-        public string[]? Groups => Group?.Split('.');
+        public string[]? Groups => Group.Split('.');
```

Comment:
`Group` can't be null on any code paths; Rider told me and I manually verified by inspecting all the constructors